centaur/src/main/resources/standardTestCases/docker_hash/docker_hash_dockerhub.wdl

```diff
@@ -3,7 +3,7 @@ task dockerhub {
         echo "hello"
     }
     runtime {
-        docker: "ubuntu:precise-20161209"
+        docker: "docker.io/ubuntu:precise-20161209"
```

Comment:
Can you add a task instead to keep testing (even though I'm sure it's tested somewhere else) the case without `docker.io` ?